.rubocop.yml

```diff
@@ -52,4 +52,4 @@ RSpec/MultipleExpectations:
   Max: 10
 
 RSpec/ExampleLength:
-  Max: 10
+  Max: 15
```

Comment:
Let's disable this warning on specific test only. If we change it here, it will apply for the whole test suite.

Reply:
solved in 7e6b0a1